### PR TITLE
Add FS

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

**Review by ndoriomedov — APPROVED** [review]

File: src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs [hunk 1]
```diff
@@ -10,10 +12,22 @@ public class InitialCreate : Migration
...
+          Name = table.Column<string>(nullable: false),
+          Extension = table.Column<string>(nullable: true),
+          Size = table.Column<int>(nullable: false),
+          Path = table.Column<string>(nullable: false),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PK_Rubrics", x => x.Id);
```
Comment (vorobevamv) [inline src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs]: Rubrics?

**Review by vorobevamv — APPROVED** [review]

File: src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs (hunk 1 above)
Comment (VLiamin) [reply src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs]: done

File: src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs [hunk 2]
```diff
@@ -10,26 +10,32 @@ namespace LT.DigitalOffice.FileService.Data.Provider.MsSql.Ef.Migrations
...
-          CreatedBy = table.Column<Guid>(nullable: false),
+      builder.CreateTable(
+        name: tableName,
+        columns: table => new
+        {
+          Id = table.Column<Guid>(nullable: false),
+          Name = table.Column<string>(nullable: false, maxLength: 245),
+          Extension = table.Column<string>(nullable: true, maxLength: 10),
+          Size = table.Column<long>(nullable: false),
+          Path = table.Column<string>(nullable: false, maxLength: 300),
```
Comment (vorobevamv) [inline src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs]: lines 26, 27, 29 - remove maxLenth

File: src/FileService.Models.Dto/Constans/DBTablesNames.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace LT.DigitalOffice.FileService.Models.Dto.Constants
+{
+  public static class DBTablesNames
+  {
+    public const string Wiki = "WikisFiles";
```
Comment (vorobevamv) [inline src/FileService.Models.Dto/Constans/DBTablesNames.cs]: WIKI

**Review by vorobevamv — APPROVED** [review]

File: src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs (hunk 2 above)
Comment (ndoriomedov) [reply src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs]: why?

Comment (VLiamin) [reply src/FileService.Data.Provider.MsSql.EF/Migrations/20220813011023_InitialCreate.cs]: done

File: src/FileService.Models.Dto/Constans/DBTablesNames.cs (hunk 3 above)
Comment (VLiamin) [reply src/FileService.Models.Dto/Constans/DBTablesNames.cs]: done

**Pull request merged**